features/storage/csi.feature

```diff
@@ -37,3 +37,17 @@ Feature: CSI testing related feature
       | /mnt/cinder/hello |
     Then the step should succeed
     And the output should contain "Hello OpenShift Storage"
+
+  # @author chaoyang@redhat.com
+  # @case_id OCP-30787
+  @admin
+  Scenario: CSI images checking in stage env
+    Given I switch to cluster admin pseudo user
+    Given I use the "csihostpath" project 
+    And pod named "my-csi-app" becomes ready
+    Given the master version >= "4.3"
+    And the expression should be true> pvc("csi-pvc").capacity(cached: false) == "2Gi"
+    Given the master version >= "4.4"
```

Comment:
I don't understand here,  if the cluseter verion < 4.4, then the case will fail?